src/main/java/org/isf/vaccine/gui/VaccineEdit.java

```diff
@@ -166,28 +161,26 @@ private JPanel getJContentPane() {
 	 */
 	private JPanel getDataPanel() {
 		if (dataPanel == null) {
-			
-			//vaccine type
-			vaccineTypeDescLabel = new JLabel();
-			vaccineTypeDescLabel.setText(MessageBundle.getMessage("angal.vaccine.vaccinetype"));
-            //vaccine code
-			codeLabel = new JLabel();
-			codeLabel.setText(MessageBundle.getMessage("angal.common.code.txt"));
+			// vaccine type
+			JLabel vaccineTypeDescLabel = new JLabel(MessageBundle.getMessage("angal.vaccine.vaccinetype") + ':');
+			// vaccine code
+			JLabel codeLabel = new JLabel(MessageBundle.getMessage("angal.common.code.txt") + ':');
 			// vaccine description
-			descLabel = new JLabel(MessageBundle.getMessage("angal.common.description.txt"));
+			JLabel descLabel = new JLabel(MessageBundle.getMessage("angal.common.description.txt") + ':');
 			// required fields
-			requiredLabel= new JLabel();
-			requiredLabel.setText(MessageBundle.getMessage("angal.vaccine.requiredfields"));
-			
+			JLabel requiredLabel = new JLabel(MessageBundle.getMessage("angal.vaccine.requiredfields"));
+
```

Comment:
It seems requiredLabel is not needed here, or can be changed to "all fields required", but if so also similar windows have same problem.

Reply:
Thanks!